Makefile

```diff
@@ -58,14 +58,15 @@ gh-preview html:
 
 gh-publish:
 	git checkout $(GH_PUBLISH_BRANCH)
-	git checkout $(GH_SOURCE_BRANCH) -- $(GH_SOURCE_DIR)
+	git rm -rf *
```

Comment:
Are untracked files a concern here? If so, we may want to do a `git clean -df` as well.